app/views/placements/provider_user_mailer/partnership_created_notification.text.erb

```diff
@@ -0,0 +1,13 @@
+<%= @user_name %>,
+
+<%= @school_name %> has added <%= @provider_name %> as one of the providers they would like to work with.
+
+##What happens next?
```

Comment:
Markdown headings need a space before the text. Some Markdown parsers are more forgiving than others, but the [GOV.UK Notify docs](https://www.notifications.service.gov.uk/using-notify/formatting#headings-and-subheadings) do say there should be a space after the hash symbols.

```suggestion
## What happens next?
```